src/main/scala/scynamo/ScynamoEncoder.scala

```diff
@@ -109,6 +110,13 @@ trait DefaultScynamoEncoderInstances extends ScynamoIterableEncoder {
   implicit val durationEncoder: ScynamoEncoder[Duration] =
     numberStringEncoder.contramap(_.toNanos.toString)
 
+  /** Using a custom formatter because
+    * "Years outside the range 0000 to 9999 must be prefixed by the plus or minus symbol."
+    * but the plus symbol is not added by the default `.toString`.
+    */
+  implicit val yearMonthEncoder: ScynamoEncoder[YearMonth] =
+    stringEncoder.contramap(_.format(DateTimeFormatter.ofPattern("uuuu-MM")))
```

Comment:
Let's extract the formatter

Reply:
Extracted in https://github.com/moia-oss/scynamo/pull/255/commits/4c49d853f644efd20b11fc415b527ba7ff0775be.